### PR TITLE
Add a soft distance to facilitate manipulation

### PR DESCRIPTION
**Pull request by frheault**
Enhancement to avoid having to dilate mask and atlas manually before filtering.

To facilitate usage it dilate everything the same way.
The enlargement is made in term of voxel, so the provided value is relative to voxel size

### CODE REVIEW

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_filter_tractogram.py [hunk 1]
```diff
@@ -96,6 +103,13 @@ def _build_arg_parser():
 def prepare_filtering_list(parser, args):
     roi_opt_list = []
     only_filtering_list = True
+
+    if args.soft_distance is not None:
+        if args.soft_distance < 1:
+            parser.error('The minimum soft distance is 1mm.')
```
Comment (arnaudbore) [inline scripts/scil_filter_tractogram.py:109]: In the help you said it was in voxel not mm

File: scripts/scil_filter_tractogram.py [hunk 2]
```diff
@@ -96,6 +103,13 @@ def _build_arg_parser():
...
+        elif args.soft_distance > 5:
+            logging.warning('Soft distance above 5mm leads to weird results.')
```
Comment (arnaudbore) [inline scripts/scil_filter_tractogram.py:111]: Same thing voxel or mm

**Review by arnaudbore — APPROVED** [review]
Small comment + it needs an update for the test_filter_tractogram.py

**Review by mdesco — APPROVED** [review]
All good after minor edits.

**Pull request merged**